Remove end of life versions

swadeley opened the pull request:
Hello

As per: 
https://access.redhat.com/support/policy/updates/satellite

we can remove some older versions.

devendra104 commented:
@swadeley The changes are already addressed in the dynaconf PR, Please check and update this PR.

```
conf/upgrade.yaml

  # Supported Satellite versions
  SUPPORTED_SAT_VERSIONS:
    - "6.7"
    - "6.8"
    - "6.9"
    - "6.10"
```

Gauravtalreja1 commented on upgrade_tests/helpers/constants.py:
```diff
@@ -7,7 +7,7 @@
 FROM_VERSION = os.environ.get('FROM_VERSION')
 TO_VERSION = os.environ.get('TO_VERSION')
-SUPPORTED_SAT_VERSIONS = ['6.2', '6.3', '6.4', '6.5', '6.6', '6.7', '6.8', '6.9']
+SUPPORTED_SAT_VERSIONS = ['6.7', '6.8', '6.9']
```
@swadeley I believe 6.10 should be appended here.

swadeley replied:
Hi @Gauravtalreja1 , I have added that.

swadeley replied:
@Gauravtalreja1 I think @devendra104 meant to say this section is obsolete due to this: https://github.com/SatelliteQE/robottelo/blob/master/conf/upgrade.yaml.template

swadeley commented:
@devendra104 I rebased and fixed merge conflict. 

I see  `from upgrade.helpers import settings` and `to_version = settings.upgrade.to_version` are new.

Please review once more.

Thank you

swadeley commented:
@devendra104 

or perhaps you mean you moved this section another repo

https://github.com/SatelliteQE/robottelo/blob/master/conf/upgrade.yaml.template
